django-stubs/utils/functional.pyi

```diff
@@ -62,8 +62,11 @@ def partition(
     predicate: Callable, values: List[_PartitionMember]
 ) -> Tuple[List[_PartitionMember], List[_PartitionMember]]: ...
 
-class classproperty:
-    fget: Optional[Callable] = ...
-    def __init__(self, method: Optional[Callable] = ...) -> None: ...
-    def __get__(self, instance: Any, cls: Optional[type] = ...) -> Any: ...
-    def getter(self, method: Callable) -> classproperty: ...
+_Get = TypeVar("_Get")
+_Self = TypeVar("_Self")
+
+class classproperty(Generic[_Get]):
+    fget: Optional[Callable[[Type[_Self]], _Get]] = ...
+    def __init__(self, method: Optional[Callable[[Type[_Self]], _Get]] = ...) -> None: ...
+    def __get__(self, instance: Any, cls: Type[_Self] = ...) -> _Get: ...
```

Comment:
From what I see `__get__` is usually an `overload` of two functions.
You can take a look here: https://github.com/python/typeshed/